tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/HotChocolateCommon.cs

```diff
@@ -142,17 +146,93 @@ private static string ConstructErrorMessage(List<IError> executionErrors)
                     if (message != null)
                     {
                         builder.AppendLine($"{tab + tab}\"message\": \"{message.Replace("\r", "\\r").Replace("\n", "\\n")}\",");
+                        eventAttributes.Add(new KeyValuePair<string, object>("message", message));
+                    }
+
+                    var locations = executionError.Locations;
+                    if (locations != null)
+                    {
+                        ConstructErrorLocationsMessage(builder, tab, locations);
+
+                        var joinedLocations = new List<string>();
```

Comment:
Wondering if it would be better to swap to a full string builder for this instead of a `List<string>`

Maybe not super important though and can always come back later

Reply:
The value of the `locations` attribute is a `string[]` so the `List<string>` looks fine to me